syncmaster/scheduler/transfer_job_manager.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+import asyncio
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from apscheduler.triggers.cron import CronTrigger
+from kombu.exceptions import KombuError
+
+from syncmaster.backend.services.unit_of_work import UnitOfWork
+from syncmaster.db.models import RunType, Status, Transfer
+from syncmaster.exceptions.run import CannotConnectToTaskQueueError
+from syncmaster.scheduler.transfer_fetcher import get_async_session
+from syncmaster.scheduler.utils import MISFIRE_GRACE_TIME
+from syncmaster.schemas.v1.connections.connection import ReadAuthDataSchema
+from syncmaster.settings import Settings
+from syncmaster.worker.config import celery
+
+
+class TransferJobManager:
+    def __init__(self, settings: Settings):
+        self.scheduler = AsyncIOScheduler(timezone=settings.TZ)
+        self.scheduler.add_jobstore("sqlalchemy", url=settings.database.sync_url)
```

Comment:
Does appscheduler handle migrations? I mean, for celery tables we manage migrations using alembic, but what about appscheduler's tables?

Reply:
It doesn't use migrations underhood, but we can use migrations if they are needed. [Added](syncmaster/db/migrations/versions/2024-11-01_0011_create_apscheduler_table.py) here, please take a look